.eslintrc.json

```diff
@@ -16,7 +16,7 @@
       },
       "ecmaVersion": 12,
       "sourceType": "module",
-      "project": ["tsconfig.json"]
+      "project": ["./ecommerce-app/tsconfig.json"]
```

Comment:
Weirdly this works completely opposite for me with nearly the same error.

Reply:
Wow I discovered something new about eslint here. I think it's because your VS Code may be opened to a folder above the actual Git project folder. According to this eslint path will by default refer to the path of the CLI calling it: https://github.com/typescript-eslint/typescript-eslint/issues/540

Reply:
That was it! Opening the editor at the .git project level then updating the path resolved this error.